Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG JENKINS_SLAVE
+FROM jenkins/jnlp-slave:${JENKINS_SLAVE}
+USER root
+
+ARG PYTHON_MAJOR
+ARG PYTHON_PATCH
+RUN export PYTHON_SEMVER="${PYTHON_MAJOR}.${PYTHON_PATCH}" && \
+  apt-get update && \
+  apt-get install wget apt-utils build-essential -y && \
+  apt-get install libreadline-dev libncursesw5-dev libssl-dev \
+  libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev libffi-dev zlib1g-dev -y && \
```

Comment:
Each packet should be in a separate line as one element, this makes future `git diff` outputs on changes much more visible